### PR TITLE
Release language-linked multimedia

### PR DESCRIPTION
**Pull request by esoergel**
Turns on this feature flag for everyone:
https://confluence.dimagi.com/display/ccinternal/Linking+multimedia+to+the+default+language
Product okay'd the release.

@proteusvacuum

### CODE REVIEW

File: corehq/apps/app_manager/models.py [hunk 1]
```diff
@@ -1453,12 +1453,11 @@ def _get_media_by_language(self, media_attr, lang, strict=False):
...
         """
         assert media_attr in ('media_image', 'media_audio')
-        toggle_enabled = toggles.LANGUAGE_LINKED_MULTIMEDIA.enabled
         app = self.get_app()
 
-        if self.use_default_image_for_all and media_attr == 'media_image' and toggle_enabled(app.domain):
+        if self.use_default_image_for_all and media_attr == 'media_image':
             lang = app.default_language
-        if self.use_default_audio_for_all and media_attr == 'media_audio' and toggle_enabled(app.domain):
+        if self.use_default_audio_for_all and media_attr == 'media_audio':
```
Comment (proteusvacuum) [inline corehq/apps/app_manager/models.py]: nit, this could probably be collapsed into a single if statement.

**Review by proteusvacuum — APPROVED** [review]

**Comment by proteusvacuum** [comment]
We should probably also move the help page out of the Solutions space, and also add a help-text link to it in the UI. Not sure if we need to update screenshots on other help pages, as this is in a pretty high-touch area.

File: corehq/apps/app_manager/models.py (hunk 1 above)
Comment (esoergel) [reply corehq/apps/app_manager/models.py]: updated

**Comment by esoergel** [comment]
> We should probably also move the help page out of the Solutions space, and also add a help-text link to it in the UI. Not sure if we need to update screenshots on other help pages, as this is in a pretty high-touch area.

I agree on the docs changes - was planning on updating that when I merge the PR.  Not convinced that this needs a help icon though - I thought it was pretty straightforward.

**Review by orangejenny — APPROVED** [review]
I vote for help icon. Nothing is straightforward for everybody.

**Comment by esoergel** [comment]
I offlined with Dev and he also thought it didn't need a help icon.  The feature is self-explanatory.  I don't think the confluence page adds much, and we shouldn't have help text for every checkbox or component in HQ.

If you want to push the issue, maybe we should take to #product

**Comment by orangejenny** [comment]
Nah, I'll defer to your & Dev's combined judgment.

**Pull request merged**